Branch-SDK-TestBed/src/main/java/io/branch/branchandroidtestbed/SettingsActivity.java

```diff
@@ -35,40 +35,21 @@ void setupRetryEditText() {
 
         retryEditText.setText(Integer.toString(currentRetries));
 
-        retryEditText.setOnEditorActionListener((textView, i, keyEvent) -> {
```

Comment:
Is this undoing your changes to set custom endpoints?

Reply:
Oh yeah it actually looks like this PR reverts some of the PR instead